lib/practice_exercises.rb

```diff
@@ -1,13 +1,49 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(m)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  (list.length - 1).times do |count|
```

Comment:
This will miss the times when the number appears 3 times in a row
For example: 
`[1, 2, 2, 3, 4]`